### ISSUE TITLE
Setting endpoint with constructor

### ISSUE THREAD

**Issue by simenandre** [cherfia/chromiumly#41]
Hello,

Thanks for this package! In our setup, we have configuration already setup. Therefore I wouldn't want to use `dotenv` files or `config`.

Is there any chance we could have the endpoint added to the constructor? Maybe even add `dotenv` and `config` as peer dependencies to get the size down as well?

**Comment by cobraz** [comment]
Having looked some more, it seems it would make sense to construct multiple instances of different modules too. Right now, it seems it has a globally scoped endpoint. If we imagine you'd want multiple Gotenberg instances, this library hinders that right now. However, seems easy to keep current functionality and get all that. Wouldn't you agree?

I don't know if that makes sense for the whole package or not, but just a though 👍

**Comment by cobraz** [comment]
To be able to bump to `node-fetch` to v3, you'll need to change to pure ESM. Maybe it's worth bumping the major version and looking at the API contracts then?

**Comment by cherfia** [comment]
@cobraz Thank you so much for your suggestions. I will include the changes in the next release 👏🏽

**Comment by danielsigl** [comment]
Hi @cherfia, could you please update docs or share a code snippet if you implemented this? Thanks in Advance!

**Comment by cherfia** [comment]
@danielsigl I haven't implemented it yet because most Node projects use either `config` or `dotenv`. However, I will try to include it since it's apparently a nice feature to have 👍

**Comment by cherfia** [comment]
@danielsigl This feature is now included in the latest release [v3.3.0](https://github.com/cherfia/chromiumly/releases/tag/v3.3.0) 🚀 

I've updated the README to include configuration via code, but here's a snippet demonstrating how it's done:

```typescript
import { Chromiumly } from "chromiumly";

Chromiumly.configure({
  endpoint: "http://localhost:3000",
  username: "user",
  password: "pass",
});
```

Please test it out and let me know if it meets your needs 👍

**Comment by danielsigl** [comment]
Hi @cherfia, thank you very much! It is working. Had to implement it like that:

```
const chromiumly = require("chromiumly");

chromiumly.Chromiumly.configure({
  endpoint: "http://localhost:3000"
});
```
